### PR TITLE
docs(v2): add section for docs-only mode

### PR DESCRIPTION
**Pull request by phoqe**
## Motivation

There's information on a Blog-only mode but there's none for a Docs-only mode. The same principle applies, and are documented in this documentation addition.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### CODE REVIEW

File: website/docs/configuration.md [hunk 1]
```diff
@@ -147,3 +147,30 @@ const Hello = () => {
...
+        docs: {
+          routeBasePath: '/' // Set this value to '/'.
+        },
+      },
+    ],
+  ],
+};
+```
+
+You can apply the same principle for blogs with the [Blog-only mode](blog.md).
```
Comment (lex111) [inline website/docs/configuration.md]:
```suggestion
You can apply the same principle for blogs with the [Blog-only mode](blog.md#blog-only-mode).
```

Comment (phoqe) [reply website/docs/configuration.md]: Ah, that's how you do it, nice

**Comment by phoqe** [comment]
@lex111 Yes, it feels like this section is a bit misplaced but there was no better place to put it since the info about the docs is a category and not a single page. Could maybe be a worthwhile mention in the setup for people that only want to use Docusaurus for the 'documentation' feature.

File: website/docs/configuration.md (hunk 1 above)
Comment (lex111) [reply website/docs/configuration.md]: Don't forget to accept (commit) this suggestion :smiley:

**Comment by lex111** [comment]
@phoqe agree, so let's wait feedback from @yangshun

**Comment by phoqe** [comment]
By the way, it appears that using only that code snippet and removing the pages altogether will results in a 404. It seems that it's required to have a index.js with a `Redirect` component.

**Pull request merged**